### PR TITLE
Improve emails design

### PR DESCRIPTION
**Pull request by desoindx**

### CODE REVIEW

File: views/emails/login.ejs [hunk 1]
```diff
@@ -1,32 +1,31 @@
...
+  </p>
+  <p>
+    <a href="<%= loginUrlWithToken %>" class="loginButton">
+      Me connecter
+    </a>
+  </p>
+  <br/>
+  <p>
+    <div>
+      Ca n'a pas marché pas en cliquant sur le bouton ?
```
Comment (sandrica89) [inline views/emails/login.ejs:19]:
```suggestion
      Ça n'a pas marché en cliquant sur le bouton ?
```

File: views/emails/login.ejs [hunk 2]
```diff
@@ -1,32 +1,31 @@
...
+    </div>
+  </p>
+  <p>
+    <a href="<%= loginUrlWithToken %>" class="loginButton">
+      Me connecter
+    </a>
+  </p>
+  <br/>
+  <p>
+    <div>
```
Comment (sandrica89) [inline views/emails/login.ejs:18]: Weird to have a `div` inside a `p`, no?

File: views/emails/loginNotAcceptedYet.ejs [hunk 3]
```diff
@@ -1,16 +1,14 @@
...
-
-
-<p>L'équipe Santé Psy Étudiant</p>
+<%- include('header'); -%>
+  <p>Bonjour ! 👋</p>
+  <p>
+    Vous avez demandé un lien de connexion à l'espace psychologues de <%= appName %>.
+  </p>
+  <p>
+    Cependant votre dossier n'a pas encore été validé par notre équipe, donc <bold>vous ne pouvez pas encore vous connecter</bold>.
```
Comment (sandrica89) [inline views/emails/loginNotAcceptedYet.ejs:7]: I didn't know this email existed :o

**Review by sandrica89 — CHANGES_REQUESTED** [review]

File: views/emails/footer.ejs [hunk 4]
```diff
@@ -0,0 +1,4 @@
+        <p>L'équipe Santé Psy Étudiant</p>
+        </div>
+    </div>
+</div>
```
Comment (sandrica89) [inline views/emails/footer.ejs:4]: No newline at the end of the file

File: views/emails/header.ejs [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+    .image {
+      width: 100%;
+    }
+</style>
+
+<div class="container">
+    <div class="card">
+      <img class="image" alt="Santé Psy Etudiant" src="https://user-images.githubusercontent.com/911434/112502968-d3011980-8d8a-11eb-9e46-f47037a6f809.png">
+      <div class="content">
```
Comment (sandrica89) [inline views/emails/header.ejs:38]: No newline at the end of the file

File: views/emails/header.ejs [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    }
+
+    .image {
+      width: 100%;
+    }
+</style>
+
+<div class="container">
+    <div class="card">
+      <img class="image" alt="Santé Psy Etudiant" src="https://user-images.githubusercontent.com/911434/112502968-d3011980-8d8a-11eb-9e46-f47037a6f809.png">
```
Comment (sandrica89) [inline views/emails/header.ejs:37]: Are you sure about the link image? :thinking:

Comment (desoindx) [reply views/emails/header.ejs:37]: it's the same as in the newletter... so yes :/

**Review by sandrica89 — APPROVED** [review]

**Pull request merged**